suite-native/graph/src/components/Graph.tsx

```diff
@@ -132,6 +185,7 @@ export const Graph = <TGraphPoint extends EnhancedGraphPoint>({
                     <GraphError error={error} onTryAgain={onTryAgain} />
                 </Box>
             )}
+            {activeTransactionEvent && <TransactionEventTooltip {...activeTransactionEvent} />}
```

Comment:
This is something I would expect to be part of library. Basically same as EventComponent is now.

Reply:
Fixed in 90955f4fc by moving Event tooltip inside of the `react-native-library`.